azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
+- 2.6.x
+- 2.9.x
```

Comment:
Note that our current active branches are master, 2.6.x and 3.x
We are no longer using 2.9.x

Tagging @sharwell

Reply:
This is primarily about getting off of Jenkins.  So 2.9.x (tracked in Jenkins) wont' ever be touched again?

Reply:
Yes, we don't intend to use that branch again.